src/test/java/trello/ui/main.java

```diff
@@ -5,41 +5,84 @@
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
-import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.junit.Assert;
 
 import java.util.List;
 
 public class main {
+
+    private static void waitForPageLoaded(WebDriver driver, long sleep) {
+        ExpectedCondition<Boolean> expectation = new
+                ExpectedCondition<Boolean>() {
+                    public Boolean apply(WebDriver driver) {
+                        return ((JavascriptExecutor) driver).executeScript("return document.readyState").toString().equals("complete");
+                    }
+                };
+        try {
+//            Thread.sleep(1000);
+            Thread.sleep(sleep);
```

Comment:
@ArturoAmoroso I think this sleep is necessary because some elements are not available after document is completely loaded. As I mentioned before using waits it's possible to avoid the sleep and also the loading page time.